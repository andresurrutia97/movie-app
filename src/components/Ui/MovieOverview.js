import React from 'react';
import PropTypes from 'prop-types';

const MovieOverview = ({ overview }) => (
  <p className="hidden md:block text-md">{overview}</p>
);

MovieOverview.propTypes = {
  overview: PropTypes.string,
};

MovieOverview.defaultProps = {
  overview: '',
};

export default MovieOverview;
