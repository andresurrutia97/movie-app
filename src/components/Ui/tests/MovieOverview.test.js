import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import MovieOverview from '../MovieOverview';

describe('<MovieOverview/>', () => {
  const component = <MovieOverview overview="text" />;

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should render a <p> tag with a text equal to "text"', () => {
    const wrapper = shallow(component);
    expect(wrapper.props().children).toBe('text');
  });
});
