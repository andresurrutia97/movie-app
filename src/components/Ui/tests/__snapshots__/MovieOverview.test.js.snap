// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MovieOverview/> should render correctly  1`] = `
<p
  className="hidden md:block text-md"
>
  text
</p>
`;
